File: back-end-server/src/Smokation.java
import java.io.Serializable;

public class Smokation implements Serializable{

	private static final long serialVersionUID = 1L;
	private double Latitude;
	private double Longitude;
	
	public Smokation(double lat, double longi){
		this.Latitude = lat;
		this.Longitude = longi;
	}

	public double getLatitude() {
		return Latitude;
	}
	public void setLatitude(double latitude) {
		Latitude = latitude;
	}
	public double getLongitude() {
		return Longitude;
	}
	public void setLongitude(double longitude) {
		Longitude = longitude;
	}

}

File: back-end-server/src/Smokation.kt
import java.io.Serializable

class Smokation : Serializable {
    var latitude: Double = 0.toDouble()
    var longitude: Double = 0.toDouble()

    constructor() {

    }

    constructor(lat: Double, longi: Double) {
        this.latitude = lat
        this.longitude = longi
    }

    @Override
    override fun toString(): String {
        return "Smokation [Latitude=$latitude, Longitude=$longitude]"
    }

    companion object {

        private val serialVersionUID = 1L
    }

}
